website/docs/reference/configs-and-properties.md

```diff
@@ -24,19 +24,22 @@ Whereas you can use **configurations** to:
 
 ## Where can I define configs?
 
-Depending on the resource type, configurations can be defined:
+Depending on the resource type, configurations can be defined in the dbt project and also in an installed package by:
 
 1. Using a [`config()` Jinja macro](/reference/dbt-jinja-functions/config) within a `model`, `snapshot`, or `test` SQL file
 2. Using a [`config` property](/reference/resource-properties/config) in a `.yml` file
 3. From the [`dbt_project.yml` file](dbt_project.yml), under the corresponding resource key (`models:`, `snapshots:`, `tests:`, etc)
 
 ### Config inheritance
 
-dbt prioritizes configurations in order of specificity, from most specificity to least specificity. This generally follows the order above: an in-file `config()` block --> properties defined in a `.yml` file --> config defined in the project file. 
+dbt prioritizes configurations in order of specificity, from most specific to least specific. This generally follows the order above: an in-file `config()` block --> properties defined in a `.yml` file --> config defined in the project file. 
```

Comment:
Sorry if this is a silly question, but if dbt "prioritizes" configs in this way does that mean they are applied this way? Meaning, is the most specific config applied and then overwritten by the least specific one? Or is the most specific one respected above the least specific?

an in-file `config()` block [ **overwrites** -->] properties defined in a `.yml` file [**overwrites** -->]  config defined in the project file. 

OR does it mean something else?

Reply:
If we say "The most specific config always takes precedence" here like we do below, this would clarify the point for me!

Reply:
i didn't author this section but i believe it is trying to explain "order of precedence" so i'm happy to update it!